SpatialGDK/Extras/schema/gameplay_debugger_component.schema

```diff
@@ -4,4 +4,5 @@ package unreal;
 component GameplayDebuggerComponent {
      id = 9997;
      uint32 delegated_virtual_worker_id = 1;
+     bool track_player = 2;
```

Comment:
This is just a nit, and probably overkill here but: Schemalang does support enums as well, you could define a matching one here too and use that instead of a bool.

Reply:
I'll leave as a bool for now, but we can extend into the full enum if we need it later.